src/Microsoft.AspNetCore.Razor.Evolution/TagHelpers/TagHelperFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.TagHelpers
+{
+    internal class TagHelperFeature : IRazorEngineFeature
```

Comment:
Because it takes in an `ITagHelperDescriptorResolver` which is currently `internal` as well. Eventually we'll have a runtime implementation of the `TagHelperDescriptorResolver` that'll need to implement `ITagHelperDescriptorResolver`. It can't do that unless the interface is public or if it's in the same project; however, we can't put the `ITagHelperDescriptoResolver` implementation in this project because it'll depend on Roslyn.